backend.py

```diff
@@ -26,15 +26,22 @@ def homepage():
             error = "Error: Word not in corpus"
     language_list = logic.SQLQuery("SELECT Lang_Desc FROM Lang_Ref;")
     part_of_speech_list = logic.SQLQuery("SELECT Part_Desc FROM Speech_Parts WHERE Lang_ID = 1;")
-    return render_template('index.html', language_list = language_list, part_of_speech_list=part_of_speech_list, sentence_List=sentence_List, error=error)
+    page_language_list = logic.SQLQuery("select Language_Page from Page_Translation;")
+    word_translated_list = logic.SQLQuery("select * from Page_Translation WHERE Language_Page = 'english';")
+    return render_template('index.html', language_list = language_list, part_of_speech_list=part_of_speech_list, sentence_List=sentence_List, error=error,page_language_list=page_language_list,word_translated_list=word_translated_list)
 
 @app.route('/Query', methods =["GET", "POST"])
 def Query():
     language_list = logic.SQLQuery("SELECT Lang_Desc FROM Lang_Ref;")
     language_selected = request.form.get('language')
     Lang_ID = logic.GetLanguageId(language_selected)
-    part_of_speech_list = logic.SQLQuery(f"SELECT Part_Desc FROM Speech_Parts WHERE Lang_ID = '{Lang_ID[0]}';")
+    part_of_speech_list = logic.SQLQuery(f"SELECT Part_Desc FROM Speech_Parts WHERE Lang_ID = '{Lang_ID}';")
```

Comment:
By removing [0] from the previous code, does that mean it will give Id's for every language now?